app/core/src/main/java/com/teamg/tourdeshot/core/api/local/domain/LocalPostDTO.java

```diff
@@ -0,0 +1,36 @@
+package com.teamg.tourdeshot.core.api.local.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Builder
+public class LocalPostDTO {
+
+    private CoordinatesDTO coordinates;
+
+    private Long ownerId;
```

Comment:
maybe move this param to path? when we enable security body should not be changing. I mean post `/local/{ownerId}